packages/client/tests/functional/extensions/__snapshots__/itx.ts.snap

```diff
@@ -65,7 +65,7 @@ exports[`extensions.itx (provider=mysql, js_planetscale) extended client in itx
 Invalid \`prisma.user.create()\` invocation:
 
 
-Unique constraint failed on the constraint: \`User_email_key\`
```

Comment:
I think that was the reason the snapshot was skipped, as not being the same as mysql?

Reply:
Yes, the MySQL snapshot uses a different logic than the Vitess+PlanetScale one